### ISSUE TITLE
https://www.habsburger.net/en

### ISSUE THREAD

**Issue by portraitsofrulers** [lovasoa/dezoomify#429]
Hi,
Can I get high resolution images from this site?
https://www.habsburger.net/en/persons/habsburg/leopold-wilhelm#o-6494

**Comment by lovasoa** [comment]
Hello,
  I don't see any zoomable image on this site. There is a traditional image, loaded all at once, that you can download by right clicking on it. 

![](https://www.habsburger.net/files/styles/large/public/originale/frans_luycx_erzherzog_leopold_wilhelm_in_geistlichem_gewand_um_1638_original.jpg?itok=lSEjd-t-)

Am I missing something?

**Comment by portraitsofrulers** [comment]
Yes, you need to reload the page until a magnifying glass appears on the photo, which enlarges the image, but I cannot understand how high the resolution of such an enlarged image is. Once upon a time, one Wikimedia user was able to get enlarged photos from this site, but did not say how he did it.

**Comment by lovasoa** [comment]
Hello,
  It looks like you can indeed get a slightly larger version of the image. After zooming in on the image with the magnifying glass, open your browser's javascript console ([firefox](https://developer.mozilla.org/en-US/docs/Tools/Browser_Console), [chrome](https://developers.google.com/web/tools/chrome-devtools/console/javascript)) and type : 

```js
document.getElementById("zcanvas").toBlob(b => window.location = URL.createObjectURL(b))
```

This will display the enlarged image, that you can then save with a right-click.


![habsburg](https://user-images.githubusercontent.com/552629/89902323-99f2ab00-dbe6-11ea-9f27-a815d6c147f4.jpg)

**Comment by portraitsofrulers** [comment]
Hello!
Thank you for your help and good luck in all your endeavors.

**Comment by AlexanderDeLargeBrotherSir** [comment]
@portraitsofrulers, please kindly close this because this does not need dezoomify at all. Thank you very kindly.
